.github/workflows/pipeline.yml

```
@@ -0,0 +1,145 @@
+          
+          # Configurar variables de entorno para la aplicación
+          cat > backend/.env << EOF
+          NODE_ENV=production
+          PORT=3010
+          DATABASE_URL=postgresql://ai4devs_user:ai4devs_password@localhost:5437/ai4devs_db
+          EOF
+          
+          # Instalar dependencias de producción
+          cd backend
+          npm ci --only=production
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)